extensions/wikia/MyHome/MyHome.class.php

```diff
@@ -179,16 +179,8 @@ public static function getInitialMainPage(Title &$title) {
 
 		//user must be logged in and have redirect enabled;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this commented out code. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS125)